addon/PlacesProvider.js

```diff
@@ -321,6 +322,12 @@ Links.prototype = {
     }
   },
 
+  getTopNewTabSites() {
```

Comment:
The only reason I'm using a promise here is because ```getData()``` explicitly needs a promise handed back to it and since ```NewTabUtils.links.getLinks()``` is sync I didn't see a point in using ```Task.async``` since a regular promise is good enough